python/tvm/script/tir/__init__.pyi

```diff
@@ -319,28 +319,28 @@ Scope handler - Loops
 
 def serial(
     begin: Union[PrimExpr, int],
-    end: Union[PrimExpr, int],
+    end: Union[PrimExpr, int] = None,
```

Comment:
Could we use overload to make it more clear?

Reply:
done